packages/backend/src/core/discovery/utils/diffToMessage.ts

```diff
@@ -1,30 +1,33 @@
 import { DiscoveryDiff } from './diffDiscovery'
 
 export function diffToMessage(name: string, diffs: DiscoveryDiff[]): string[] {
```

Comment:
I was thinking more about something like this:
```
export function diffToMessage(name: string, diffs: DiscoveryDiff[]): string[] {
  const header = wrapBold(`Detected changes for ${name}`)
  const messages: string[][] = [[]]
  let index = 0

  for (const diff of diffs) {
    const currentLength = header.length + wrapCodeBlock(messages[index]).length
    const nextDiff = diffToString(diff)

    if (currentLength + nextDiff.length >= 2000) {
      index += 1
      messages[index] = []
    }

    messages[index].push(nextDiff)
  }

  const result = messages.map((m) => wrapCodeBlock(m))
  return result
}

export function wrapBold(message: string) {
  return `**${message}**`
}

export function wrapCodeBlock(messages: string[]) {
  const prefix = '```\n'
  const postfix = '\n```'

  return `${prefix}${messages.join('\n')}${postfix}`
}
```

Reply:
We can get on call and talk about this 👍

Reply:
I wanted to wrap all the cahnges connected with a project into one block of code, I think it would be more readable, WDYT?